src/otx/config/model/mmaction_detection.yaml

```diff
@@ -5,7 +5,7 @@ _target_: otx.core.model.module.action_detection.OTXActionDetLitModule
 
 otx_model:
   _target_: otx.core.model.entity.action_detection.MMActionCompatibleModel
-  config: ???
+  num_classes: ???
```

Comment:
Should TopK be added here?

Reply:
If every MMAction model should configure `top_k`, then please add it there.